src/Resources/config/services.yml

```diff
@@ -27,6 +27,18 @@ services:
         tags:
             - { name: kernel.cache_warmer }
 
+    contao.controller.abstract_csv_import:
```

Comment:
I would not create an abstract service for this just because there's an abstract class. It's pretty unlikely that the abstract service will ever be used again.